05-visualisation-ggplot2.Rmd

```diff
@@ -285,4 +285,36 @@ ggplot(data = yearly_weight, aes(x=year, y=avg_weight, color = species_id, group
 ```{r average-weight-time-facet_sex}
 ggplot(data = yearly_weight, aes(x=year, y=avg_weight, color = species_id, group = species_id)) +
   geom_line() + facet_wrap(~ sex)
-```
\ No newline at end of file
+```
+
+So far our result looks quite good, but it is yet far from being publishable. What are other ways one can improve it? Take a look at the ggplot2 cheat sheet (https://www.rstudio.com/wp-content/uploads/2015/08/ggplot2-cheatsheet.pdf), and write down at least three more ideas (can leave them as comments to Etherpad.
```

Comment:
* missing closing parenthesis after Etherpad.
* (**you** can leave them as comments in the Etherpad)